src/modules/submit.js

```diff
@@ -0,0 +1,30 @@
+const nameInput = document.getElementById('name-input');
+const scoreInput = document.getElementById('score-input');
+
+const submit = (e) => {
+  e.preventDefault();
+  if (nameInput.value && scoreInput.value) {
+    const data = {
+      user: nameInput.value,
+      score: scoreInput.value,
+    };
+    fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/HW6Rm5sf6ZsLRuJNGtLy/scores/', {
+      method: 'POST', // or 'PUT'
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        console.log('Success:', data);
+      })
+      .catch((error) => {
+        console.error('Error:', error);
+      });
+    nameInput.value = '';
+    scoreInput.value = '';
+  }
+};
```

Comment:
- [ ] As I mentioned above :point_up:  please use `async` and `await`  instead of  `.then()` and `.catch()` and reflector your code and also check my suggestion below :point_down: 

```suggestion
const submit = async (e) => {
  e.preventDefault();
  if (nameInput.value && scoreInput.value) {
    const data = {
      user: nameInput.value,
      score: scoreInput.value,
    };
    try {
      const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/HW6Rm5sf6ZsLRuJNGtLy/scores/', {
        method: 'POST', // or 'PUT'
        headers: {
          'Content-Type': 'application/json',
        },
        body: JSON.stringify(data),
      });
      const result = await response.json();
      console.log('Success:', result);
    } catch (error) {
      console.error('Error:', error);
    }
    nameInput.value = '';
    scoreInput.value = '';
  }
};
```

- [ ] Similar to the previous code :point_up: , we use `async/await` to handle the asynchronous `fetch` request. We also use a `try...catch` block to handle any potential errors.

- [ ] Instead of chaining multiple `.then()` and `.catch()` methods, we can use `await` to retrieve the response object and parse the response data with `response.json()` directly.

- [ ] Finally, we clear the input fields outside of the `try...catch` block since it is a synchronous operation that does not depend on the API response.